Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1359,7 +1359,7 @@ def gen2018HiMix(fragment,howMuch):
 #for 2015
 step3Up2015Defaults = {
     #'-s':'RAW2DIGI,L1Reco,RECO,EI,VALIDATION,DQM',
-    '-s':'RAW2DIGI,L1Reco,RECO,EI,PAT,VALIDATION:@standardValidation+@miniAODValidation,DQM:@standardDQM+@miniAODDQM',
+    '-s':'RAW2DIGI,L1Reco,RECO,RECOSIM,EI,PAT,VALIDATION:@standardValidation+@miniAODValidation,DQM:@standardDQM+@miniAODDQM',
```

Comment:
should this be added to the run1 workflows as well?

Reply:
I think it's good enough at the moment.